tests/bsim/bluetooth/audio/sysbuild.cmake

```diff
@@ -20,6 +20,7 @@ if(SB_CONFIG_NET_CORE_IMAGE_HCI_IPC)
 
 	set(${NET_APP}_EXTRA_CONF_FILE
 	 ${APP_DIR}/overlay-nrf5340_cpunet_iso-bt_ll_sw_split.conf
+	 ${NET_APP_SRC_DIR}/debug_overlay.conf
```

Comment:
Was not aware of this file. It's kind of weird that we've had this file for 3 years, and there's no mention or references (or uses of it) until now. Seems like something that could/should be reference in the hci_ipc sample's README. cc @MarekPieta